source/install/desktop-msi-installer-and-group-policy-install.rst

```diff
@@ -66,66 +66,66 @@ Installing group policy files locally
+2. In the ``Edit group policy`` window, navigate to ``Local Computer Policy\Computer Configuration\Administrative Templates\Mattermost``. In this example, double-click on ``DefaultServerList`` to set one or more default servers that will appear on app launch. 
 
    .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00010.png
 
-3. In the resulting window for ``DefaultServerList``, click on ``Enabled`` to turn the feature on and then click on the ``Show…`` button in the ``Options:`` section of the window to add default servers.
+3. In the resulting window for ``DefaultServerList``, select **Enabled** to turn the feature on, then select the **Show…** button in the **Options:** section of the window to add default servers.
 
    .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00011.png
 
-4. In the resulting window, add desired Mattermost servers using a memorable name (i.e. Community) and the web url of the Mattermost server (i.e. https://community.mattermost.com).
+4. In the resulting window, add desired Mattermost servers using a memorable name (i.e., Community) and the web URL of the Mattermost server (i.e., https://community.mattermost.com).
 
-5. Click the ``OK`` button twice and close the ``Edit group policy`` application.
+5. Select **OK** twice, then close the **Edit group policy** app.
 
    .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00012.png
 
 Verify group policy settings have been applied
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1. To verify the settings applied correctly, open up the ``Registry Editor`` by clicking on the ``Start`` button, typing ``Registry Editor`` in the search field and selecting the ``Registry Editor`` option in the search results.
+1. Open up the ``Registry Editor`` by selecting **Start**, typing ``Registry Editor`` in the search field, then selecting the **Registry Editor** option in the search results.
 
    .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00013.png
 
-2. In the ``Registry Editor`` window, navigate to ``Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Mattermost\DefaultServerList`` and verify the servers you added using the ``Edit group policy`` app are listed.
+2. In the **Registry Editor** window, navigate to ``Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Mattermost\DefaultServerList`` and verify the servers you added using the **Edit group policy** app are listed.
 
-3. Close the ``Registry Editor`` once verified.
+3. Once verified, close the **Registry Editor**.
 
    .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00014.png
 
 Install Mattermost Desktop using the MSI installer
```

Comment:
Is this a different app to the Desktop App?

Reply:
Great catch! Fixed.